pkg/cn_controller/cn_pod.go

```diff
@@ -22,6 +22,12 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"strconv"
+)
+
+const (
+	log_path = "/opt/starrocks/be/log"
```

Comment:
is it better to give `cn` a different path than `be`?

Reply:
the log path is starrocks be set, operator is only follow the rule.

Reply:
it is possible to set to `cn/log`

in dockerfile, copy `output/be` to `/opt/starrocks/cn` , set STARROCKS_ROOT=/opt/starrocks, set STARROCKS_HOME=${STARROCKS_ROOT}/cn